### DISCUSSION TITLE
Clear recent opened files

### DISCUSSION THREAD

**Discussion by Dominiquini** [SubtitleEdit/subtitleedit#4962] [Ideas]
Add a option to clear the list of recently opened files. 

Thanks

**Comment by OmrSi** [comment]
Applied in https://github.com/SubtitleEdit/subtitleedit/pull/4963

**Comment by OmrSi** [comment]
You can now use this in the latest beta https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.0/SubtitleEditBeta.zip
Will also be available with the next official release.

**Reply by Dominiquini** [reply]
I found a bug!

1) Open a .srt file with Subtitle Edit (double click on the file)
2) Clear the file history (File -> Reopen -> Clear)
3) Close Subtitle Edit
4) I open Subtitle Edit again from the Windows start menu
5) The program opens with the last subtitle loaded! 

Thanks.

**Reply by OmrSi** [reply]
I'm not sure much can be done about that. SE saves the opened file when closing so it can be opened again when opening SE.

**Reply by Dominiquini** [reply]
Thank you anyway!
Great project that you are working on!
